tests/std/tests/P1614R2_spaceship/test.cpp

```diff
@@ -178,6 +178,27 @@ void unordered_containers_test(
     assert(something != different);
 }
 
+template <class Iter, class ConstIter>
+void ordered_iterator_test(const Iter& smaller, const Iter& smaller_equal, const Iter& larger,
+    const ConstIter& const_smaller, const ConstIter& const_smaller_equal, const ConstIter& const_larger) {
+    spaceship_test<std::strong_ordering>(smaller, smaller_equal, larger);
+    spaceship_test<std::strong_ordering>(const_smaller, const_smaller_equal, const_larger);
+    spaceship_test<std::strong_ordering>(const_smaller, smaller_equal, larger);
+}
+
+template <class Iter, class ConstIter>
+void unordered_iterator_test(const Iter& something, const Iter& something_equal, const Iter& different,

```

Comment:
I'm not sure these `unordered_iterator_test` cases belong here, or aren't redundant with coverage elsewhere, but I suppose they aren't hurting anything.

Reply:
This exercises the constness scenarios so I believe they are valuable.

Reply:
Good enough for me.